app/lib/shared/handlers.dart

```diff
@@ -54,11 +54,30 @@ shelf.Response jsonResponse(
   );
 }
 
+final _contentSecurityPolicyMap = <String, List<String>>{
+  'frame-src': <String>["'self'"],
+  'script-src': <String>[
+    "'self'",
+    "'unsafe-eval'", // dart2js requires it
```

Comment:
Let's file an issue for this... and put in the backlog, there is no reason to rush to doing it... but it's a nice thing to fix on a rainy day.

Reply:
I think that the referenced issue just talks about a test that is written incorrectly, so it shouldn't affect anything except that particular test.